.github/workflows/post-commit.yml

```diff
@@ -293,7 +293,10 @@ jobs:
             maven-repo-
       - name: Execute tests
         run: hadoop-ozone/dev-support/checks/integration.sh -P${{ matrix.profile }}
-        continue-on-error: ${{ matrix.profile == 'flaky' }}
+        if: matrix.profile != 'flaky'
+      - name: Execute flaky tests
+        run: hadoop-ozone/dev-support/checks/integration.sh -P${{ matrix.profile }} -Dsurefire.rerunFailingTestsCount=5 -Dsurefire.fork.timeout=3600
```

Comment:
Very nice!  I didn't know about -Dsurefire.rerunFailingTestsCount

Reply:
> So what happens if the test fails 5 times? does the whole run fail?

Yes, it fails in that case.  It's really 5 reruns, so 6 attempts total.